plugins/inputs/mikrotik/mikrotik.go

```diff
@@ -0,0 +1,227 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package mikrotik
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"io"
+	"net/http"
+	"time"
+
+	common_tls "github.com/influxdata/telegraf/plugins/common/tls"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+type Mikrotik struct {
+	Address         string          `toml:"address"`
+	IgnoreCert      bool            `toml:"ignore_cert,omitempty"`
+	ResponseTimeout config.Duration `toml:"response_timeout"`
+	IgnoreComments  []string        `toml:"ignore_comments"`
+	IncludeModules  []string        `toml:"include_modules"`
+	Username        string          `toml:"username"`
```

Comment:
Please also make the `Username` a `config.Secret` so people can fill credentials from secret-stores if they need to.